lib/ex_prosemirror/config.ex

```diff
@@ -55,6 +56,13 @@ defmodule ExProsemirror.Config do
     config
     |> do_override(input_config, :marks)
     |> do_override(input_config, :blocks)
+    |> do_override(input_config, :inline)
+  end
+
+  defp do_override(config, input_config, :inline) do
+    if is_nil(Keyword.get(input_config, :inline)),
+      do: config,
+      else: Keyword.put(config, :inline, Keyword.get(input_config, :inline))
```

Comment:
```suggestion
  defp do_override(config, input_config, :inline) do
    if inline_opts = Keyword.get(input_config, :inline) do
      Keyword.put(config, :inline, inline_opts)
    else
      config
```

Because you don't need to call twice `Keyword.get/2`